src/modules/Packages.rb

```diff
@@ -1515,6 +1515,20 @@ def Initialize_BaseInit(show_popup, base_url, log_url)
       true
     end
 
+    # Fallback repository name
+    # @return [String] translated name
+    def fallback_name
+      _("Repository")
```

Comment:
is it really good idea to have it localized? Other repos are also not localized, not?

Reply:
I'm not sure, but normally the fallback should not be used...